week1/3-website/index.html

```diff
@@ -16,7 +17,90 @@
     <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
   </head>
   <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+      <header id="header">
+        
+        <h1>The importance of good sleep</h1>
+        <!--<img id="header-img" src="images/sleepingCat.jpeg"      height: 90px;    width: 50px;  alt="a sleeping cat" />-->
+        <p>Sleep is essential to the maintenance of physical and psychological health.<br>  We all know how difficult life can be when we’re not sleeping well.<br>  Poor sleep can have a negative impact on your energy levels, mood, performance and enjoyment of life. <br> So it’s worth taking a few minutes to think about how you might be able to improve your quality and quantity of sleep.</p>
+        <nav>
+          <ul>
+            <li><a class="nav-link" href="#tips">Top tips to sleep better</a></li>
+            
+            <li><a class="nav-link" href="#calories">How Body Uses Calories While Sleep</a></li>
+
+            <li><a class="nav-link" href="#science">What the Science Says</a></li>
+            
+         </ul>
+        </nav>
+       
+
+      </header>
+      <div class="content">
+      <main>
+         <img id="header-img" src="brainWhileSleeping.jpg"        alt="a drawing showing the toxing cleaning while asleep" />
+         
+        <section>
+         <article id="tips" class="article" >
```

Comment:
I like the ID system you use for these, but having class 'article' for the article element is a bit redundant!